yarn-project/foundation/src/json-rpc/server/json_rpc_server.ts

```diff
@@ -21,6 +21,7 @@ export class JsonRpcServer {
     stringClassMap: StringClassConverterInput,
     objectClassMap: JsonClassConverterInput,
     private createApi: boolean,
+    private createStatusEndpoint: boolean,
```

Comment:
This seems a bit hacky. Can't we just add a route outside of this?

Reply:
You think? I felt that would become useful as we're deploying things now. 
Can do for sure within the sandbox code that starts the node